src/PerformanceTests/Tests/Statistics.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
```

Comment:
Why is this moved to the test project? It already is becoming the root of all evil. Stuff should be moved out of it.